helix-core/src/main/java/org/apache/helix/HelixCloudProperty.java

```diff
@@ -109,6 +116,9 @@ public void populateFieldsWithCloudConfig(CloudConfig cloudConfig) {
           setCloudInfoSources(
               Collections.singletonList(azureProperties.getProperty(CLOUD_INFO_SOURCE)));
           setCloudInfoProcessorName(azureProperties.getProperty(CLOUD_INFO_PROCESSOR_NAME));
+          setCloudInfoProcessorFullyQualifiedClassName(
```

Comment:
NIT: Format problem? Helix has our own style formatter.

Reply:
I reformatted the code, with helix style. The formating here should have been from that as well based on my intellij settings.